src/image_processing_effects.c

```diff
@@ -761,211 +761,53 @@ static u16 QuantizePixel_BlurHard(u16 *prevPixel, u16 *curPixel, u16 *nextPixel)
     return RGB2(red, green, blue);
 }
 
-/*
-void ConvertImageProcessingToGBA(struct ImageProcessingContext *arg0)
+void ConvertImageProcessingToGBA(struct ImageProcessingContext *context)
 {
     u16 i, j, k;
-    u8 r5 = arg0->canvasWidth >> 3;
-    u8 var_24 = arg0->canvasHeight >> 3;
-    u16 (*var_2C)[][32] = arg0->canvasPixels;
-    u32 var_28 = arg0->dest;
+    u16 *src, *dest, *src_, *dest_;
+    u16 width, height;
 
-    if (arg0->var_16 == 2)
+    width = context->canvasWidth >> 3;
+    height = context->canvasHeight >> 3;
+    src_ = context->canvasPixels;
+    dest_ = context->dest;
+
+    if (context->var_16 == 2)
```

Comment:
What other values can var_16 take?

Reply:
var_16 is set up only once in the entire game in contest_painting.c to value 2. So no clue what it stands for.

Reply:
Seems to be an indicator for whether the source image is unpacked 8bpp or unpacked 4bpp 🤔